packages/tokens/src/core.json

```diff
@@ -1058,32 +1058,34 @@
           "paddingTop": {
             "value": "{space.xl}",
             "type": "spacing",
-            "description": "space.button.solid.block.md"
+            "description": "space.dialog.desktop.container.paddingTop"
```

Comment:
should this not be "blockStart"?

Reply:
Will delay this rename until contextual tokens.